code/src/cljs/sixsq/nuvla/ui/common_components/job/views.cljs

```diff
@@ -44,6 +44,7 @@
 
 (defn JobsTable
   [_jobs]
+  (dispatch [::pagination-plugin/change-page [::spec/pagination] 1])
```

Comment:
Did you consider placing this inside the `Tab` component as a `change-event`? like we do [here](https://github.com/nuvla/ui/blob/master/code/src/cljs/sixsq/nuvla/ui/pages/apps/apps_store/views.cljs#L181). So that it gets dispatched anyway whenever you switch tabs (if I understand correctly) - maybe it's useful for other views too 🤷‍♀️

Reply:
Yes, to avoid the developer to remember to do it when he use the component, I preferred to take care of it within the component